docs/scaling_schedules_crd.yaml

```diff
@@ -0,0 +1,118 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.5.0
+  creationTimestamp: null
+  name: scalingschedules.zalando.org
+spec:
+  group: zalando.org
+  names:
+    kind: ScalingSchedule
+    listKind: ScalingScheduleList
+    plural: scalingschedules
+    singular: scalingschedule
+  scope: Namespaced
```

Comment:
@mikkeloscar regarding our last conversation: after a closer look my idea is to go with just one namespace CRD and make it possible for HPAs to define the object namespace if not the default. The default isn't `default`, I would consider, in the implementation, the same namespace as the HPA itself. Just like name resolution for services. What do you think? Is it feasible? Do you see drawbacks?

Reply:
The problem here is that you can't specify the `namespace` in the HPA object reference: https://github.com/kubernetes/kubernetes/blob/cf3374e43491c594071548f68d4357fc9ed537ea/staging/src/k8s.io/api/autoscaling/v2beta2/types.go#L86-L94

So unless we would encode it in the name (or use an external metric type to reference it) I don't know how to get around defining both a namespaced and cluster spaced resource type if we want to support both use cases.

Reply:
Thanks for the context, you're right, we will have to support both, which shouldn't be a problem. Will fix.